docs/source/conf.py

```diff
@@ -240,11 +240,11 @@
     "https://docs.github.com/en/rest/overview/other-authentication-methods#via-username-and-password",
     "https://www.educative.io/blog/advanced-yaml-syntax-cheatsheet#anchors",
     # temporarily, until the file actually exists in `main`
-    "https://github.com/kedro-org/kedro/blob/main/kedro/templates/project/%7B%7B%20cookiecutter.repo_name%20%7D%7D/.flake8",
+    "https://www.quora.com/What-is-thread-safety-in-Python",  # "403 Client Error: Forbidden for url"
```

Comment:
Not sure what to think about this though, the page does exist... https://www.quora.com/What-is-thread-safety-in-Python

I suspect Quora is just rate-limiting RTD